src/rustdoc-internals.md

```diff
@@ -201,7 +201,7 @@ server. To test these features locally, you can run a local HTTP server, like
 this:
 
 ```bash
-$ ./x.py doc library/std
+$ ./x.py doc library/std --stage 1
```

Comment:
```suggestion
$ ./x.py doc library/std --stage 1  # defaults to stage 0
```

Reply:
This is already documented: https://rustc-dev-guide.rust-lang.org/building/bootstrapping.html#default-stages

Reply:
Ah, I didn't saw that, I read rustdoc directly such that I skipped that part.